.github/workflows/deploy_branches.yml

```diff
@@ -24,12 +24,13 @@ jobs:
           key: ${{ runner.os }}-modules-${{ hashFiles('./source/opam.export') }}
       - name: Install dependencies and build hazel # STEP 3
         run: |
-          sudo apt --assume-yes install curl m4 opam
+          sudo apt --assume-yes install curl m4 opam nodejs
```

Comment:
It might be worth using https://github.com/actions/setup-node in a previous step so that we're not accidentally upgrading the node version with Ubuntu. I could take it or leave it though.